dhis-2/build-dev.sh

```diff
@@ -23,6 +24,16 @@ TAG=local
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
+# Choosing this approach over automatically activating a maven profile based on the architecture.
+# This is to not risk running both the jibBuild and jibBuildArmOnly profiles in our pipelines.
+# There might be ways like using https://maven.apache.org/enforcer/enforcer-rules/requireActiveProfile.html
+# to prevent that but they would require more work.
+ARCH=$(mvn help:system | grep "os\.arch")
+JIB_PROFILE=
+if [[ "$ARCH" == *arm64* ]]; then
```

Comment:
`os.arch` might be something like `aarch64` on some hosts. Someone with an M1 Mac should test this.

Reply:
> os.arch might be something like aarch64 on some hosts. Someone with an M1 Mac should test this.

We should and will definitely test this! 😄 

We are using `mvn` to emulate what we would do when activating profiles automatically via https://maven.apache.org/guides/introduction/introduction-to-profiles.html

I opted for it since I think it does a little "sanitizing" for us. I cannot find all the arch values (yet).

Maven profile code uses
https://github.com/codehaus-plexus/plexus-utils/blob/4f333f2691873e5ba024991a03390a4e773511e9/src/main/java/org/codehaus/plexus/util/Os.java#L103

For example for me I get

```sh
uname -m
x86_64

mvn help:system | grep "os\.arch"
os.arch=amd64
```

Reply:
Lets test for `arm64` and `aarch64` to be safe. Both refer to the same architecture and seem to be used interchangeably.